_data/github_details.yml

```diff
@@ -0,0 +1,47 @@
+- rank: 1
+  team: 6
+  username1: Sai Keshari
+  username2: Neha Kumari
+  username1git: https://github.com/saikeshari
+  username2git: https://github.com/neha-93
+  PRs: 1
+
+- rank: 2
+  team: 1
+  username1: Yuheng Wu
+  username2: Abdulrahman Yusuf
+  username1git: https://github.com/iamwyh2019
+  username2git: https://github.com/Young-Einstein10
+  PRs: 0
+
+- rank: 3
+  team: 2
+  username1: Ojuswi Rastogi
+  username2: Gaurav Kumar
+  username1git: https://github.com/OjusWiZard
+  username2git: https://github.com/GrayFlash
+  PRs: 0
+
+- rank: 4
+  team: 3
+  username1: Adarsh Naidu
```

Comment:
Do we still need this since you are no longer hardcoding? Please, remove if no longer needed